suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -18,36 +18,43 @@ import {
 import { selectFiatCurrency } from '@suite-native/module-settings';
 import { useFormatters } from '@suite-common/formatters';
 
-type AccountTransactionListItemProps = {
+import { TransactionListItemIcon } from './TransactionListItemIcon';
+
+type TransactionListItemProps = {
     transaction: WalletAccountTransaction;
 };
 
-const transactionIconMap: Partial<Record<TransactionType, IconName>> = {
-    recv: 'receive',
-    sent: 'send',
+type TransactionTypeProperties = {
+    prefix: 'From' | 'To';
+    sign: '+' | '-';
+    signColor: keyof Colors;
 };
-
-const transactionTextMap: Partial<Record<TransactionType, string>> = {
-    recv: 'Received',
-    sent: 'Sent',
+const transactionTypePropertiesMap: Partial<Record<TransactionType, TransactionTypeProperties>> = {
```

Comment:
Can you please cover all cases: `type: 'sent' | 'recv' | 'self' | 'joint' | 'failed' | 'unknown';` I guess there won't be sign for most of them, but at least you can give them better name (or easily change name in future, which will be requested for sure)

Reply:
fixed [30b5f1a](https://github.com/trezor/trezor-suite/pull/7392/commits/30b5f1a27811ed560da7091928ce739d439d0ca9)